packages/webpack/src/buildtime/emitSes.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+const { readFileSync } = require('fs')
+const {
+  sources: { RawSource },
+} = require('webpack')
+
+module.exports = {
+  sesEmitHook:
+    ({ compilation, HtmlWebpackPluginInUse, HtmlWebpackPluginInterop }) =>
+    () => {
+      // Read the file you want to add to the output
+      const sesFile = readFileSync(require.resolve('ses'), 'utf-8')
+      const asset = new RawSource(sesFile)
+
+      compilation.emitAsset('lockdown.js', asset)
+
+      if (HtmlWebpackPluginInUse && HtmlWebpackPluginInterop) {
+        HtmlWebpackPluginInUse.constructor
+          .getHooks(compilation)
+          .beforeEmit.tapAsync('LavaMoatWebpackPlugin-lockdown', (data, cb) => {
+            const scriptTag = '<script src="./lockdown.js"></script>'
+            const headTagRegex = /<head[^>]*>/i
+            const scriptTagRegex = /<script/i
```

Comment:
Agree on the regex+html being impossible to fix. There's a long tail of cases where it could break. 
I'm willing to accept that option since it's a convenience helper and the default remains that you include lockdown at your own accord.